shells/pipes-shell/web/deploy/android_deploy.sh

```diff
@@ -6,10 +6,18 @@ OUT_DIR="$PWD/$1"
 # cd to directory containing this script.
 cd "${0%/*}"
 
+# Webpack worker code.
+# Must follow the physical directory instead of symbolic link otherwise
+# webpack cannot resolve the relative paths at build/** or shells/lib/build/**
+CURR_SYMLINK_DIR=$PWD
+cd -P ../../../..
+tools/sigh --quiet webpack
```

Comment:
Instead of running this command manually in the script, could you update the `//shells/pipes-shell:pipes_shell_android` BUILD rule and add a dep on `//tools:sigh_webpack`?

That will make bazel run the webpack script when needed. And it lets us consolidate the requirements for running webpack in the one place (so if the args or deps for that change, we only have to change it in the one place).

Reply:
I actually did this at the very first place by add "//tools:sigh_webpack" to deps but it didn't work out at a "clean" build where `./tools/sigh clean and bazel clean --expunge`
The root cause is the build/** is generated at bazel .cache folder instead of <repo root>/build so that the references to tsc'd js sources cannot be linked during webpacking as expected.
This is also why I add the magic "cd -P" to let it work at a clean build.

Reply:
So one way-out is to modify the script.
If you prefer to changing in a one place for the best maintenance, 
I do have a solution which just modifies the sigh_command
-cmd = "$(location //tools:sigh_bin) "
+cmd = "cd -P src/..; $(location //tools:sigh_bin) "

This does all tricks for us at both of clean and non-clean builds.
Does this sound better to you?

Reply:
Oh right, good catch with the bazel cache folder. That's a pity.

Your modification to the sigh_command sounds good to me! Could you add a comment there too explaining what it's doing? Thanks!

Reply:
Great! Done!